### DISCUSSION TITLE
Trigger the dropdown menu through field in the Autocomplete component

### DISCUSSION THREAD

**Discussion by MightyFo0l** [primefaces/community#3247] [PrimeNG]
Hi, how can I make sure that the options get shown when the field gets active, not just through clicking on the chevron button?

[https://primeng.org/autocomplete#dropdown](https://primeng.org/autocomplete#dropdown)

In the 'Dropdown' example, you need to add `[dropdown]="true"` to make the chevron (and thus the menu) appear. But it would be ideal if the menu can appear when activating the field. I think that makes for a superior User Experience.

Thank you!

**Comment by MightyFo0l** [comment]
Bump

**Comment by benoitjry** [comment]
I agree, I would like to be able to see the dropdown on focus if my minLenght is 0 (for example). Other idea is put dropdown mode on the field but i think it's make less sense.
